CalPatRec/src/CalHelixFinderAlg.cc

```diff
@@ -1278,9 +1278,9 @@ namespace mu2e {
 // gianipez: procedure for aligning the phi vector
 //-----------------------------------------------------------------------------
     ::LsqSums4        szphi;
-    int               count(0);
-    float            chi2min, deltaPhi, dphi_max(0);
-
+    int               count(0), minNFitHits(5);
+    float             chi2min, deltaPhi, dphi_max(0);
```

Comment:
Please make minNFitHits constexpr.

This file already has another variable, _minHHits.  Is there a good reason to make a new one?

Reply:
No problem.
the data member is used for a different purpose; `_minHits` is used to select at the end of the search if a helix has enough hits, while the new I introduced is just a threshold for updating the fit results.

Reply:
When you make the change, please add a comment to distinguish it from _minHits.